jwst/master_background/master_background_mos_step.py

```diff
@@ -106,9 +107,10 @@ def process(self, data):
             # If some type of background processing had already been done. Abort.
             # UNLESS forcing is enacted.
             if not self.force_subtract and \
-               'COMPLETE' in [data_model.meta.cal_step.back_sub, data_model.meta.cal_step.master_background]:
+               'COMPLETE' in [query_step_status_container(data_model, "back_sub"), \
```

Comment:
The replaced code:
```
COMPLETE' in [data_model.meta.cal_step.back_sub, data_model.meta.cal_step.master_background]:
```
Did not index into a container. Is `data_model` ever a `ModelContainer`? The step only mentions `MultiSlitModel` and the one use I can find in `calwebb_spec2` looks like it should be passing a model into this step:
https://github.com/spacetelescope/jwst/blob/ce9a135f3bc6bba21e95dd47a91fc4d9ba5a1991/jwst/pipeline/calwebb_spec2.py#L548

Reply:
yes, there are some instances where both of these are passed a ModelContainer in the regtest suite.  I can track those down for you if you want